### ISSUE TITLE
jQuery Bridget in ES6 modules

### ISSUE THREAD

**Issue by trajektorijus** [desandro/jquery-bridget#10]
Is this possible?
I tried running it like so:

`import $ from "jquery";`
`import Bridget from "desandro/jquery-bridget";`
`import Isotope from "isotope-layout";`
`console.log(Bridget);`

The result is empty object... Did I do something wrong? Do I need to shim it somehow?

I am using JSPM for module management and build.

**Comment by trajektorijus** [comment]
Also, I am not sure if it's related, but when I try to install jquery-bridget through the npm I get:
`err  Installing npm:jquery-bridget@2.0.0, no version match for npm:jquery@^3`

**Comment by trajektorijus** [comment]
Ok, I solved this issue. I think it's a JSPM bug after all...
What I did:
1. Installed jquery-bridget through jspm & github like so: `jspm install github:desandro/jquery-bridget`
2. This resulted in an empty object described in the first post.
3. Removed jquery-bridget: `jspm remove github:desandro/jquery-bridget`
4. Tried installing the npm one: `jspm install npm:jquery-bridget`
5. Got the error in my second post.
6. Removed jquery-bridget: `jspm remove npm:jquery-bridget`
7. Installed jquery-bridget through npm directly: `npm install jquery-bridget`
8. Added this `"jquery-bridget": "npm:jquery-bridget@2.0.0"` to my jspm_config map prop.
